packages/web-components/src/components/carousel/carousel.ts

```diff
@@ -111,6 +114,35 @@ class DDSCarousel extends LitElement {
     }
   }
 
+  /**
+   * Handles card focus throughout pages.
+   *
+   * @param event The event.
+   */
+  @HostListener('shadowRoot:focusin')
+  // @ts-ignore: The decorator refers to this method but TS thinks this method is not referred to
+  private _handleFocus = async ({ target, relatedTarget }: FocusEvent) => {
+    const currentContains = target !== this && this.contains(target as DDSCard);
+    const oldContains = target !== this && !this.contains(relatedTarget as DDSCard);
+    const currentCardIndex = Array.from(this.children).indexOf(target as HTMLElement);
+
+    // reset to first page if tabbing back into the carousel after previously moving pages
+    if (currentContains && oldContains) {
+      this.start = 0;
```

Comment:
1. Did you really want to run the next `if ()` block in this condition, or did you want to bail out from `_handleFocus()` in this case?
2. What should happen in back-tab scenario? (e.g. When the focus was on the previous page button (pagination button with left caret icon) and user hits <kbd>Shift</kbd>+<kbd>Tab</kbd> key)?
3. If user was previously on the non-first page and then reaches to carousel in tab key, what should happen? (e.g. User goes to the 3rd page by clicking on next page button (pagination button with right caret icon) two times, clicks on the blank area above the carousel and then hit tab key)